pyproject.toml

```diff
@@ -19,11 +19,12 @@ azure-storage-blob = { version = "^12.0.0", optional = true }
 cffi = {version = "^1.14.0", optional = true}
 
 [tool.poetry.dev-dependencies]
-pytest = "^5.4.3"
+pytest = "^6.0.0"
```

Comment:
This may cause problems, as many of our internal products require pytest to version 5. Would it be OK to upgrade them as well?